babyhandling.html

```diff
@@ -5,72 +5,188 @@
+        <img
+          class="rounded-lg pb-8"
+          src="./h1baby.jpg"
+          alt="Baby feet in a soft white blanket"
+        />
+      </div>
+      <div class="space-y-3">
+        <h2 class="text-blue-800 text-3xl font-semibold">
+          Was ist Baby Handling?
+        </h2>
+        <p class="font-sans text-gray-800">
+          Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ad libero
+          saepe atque, reiciendis fuga, ratione a omnis iusto nam quasi,
+          consectetur facilis voluptatum quas ullam repudiandae assumenda
+          tempore animi nemo? Lorem ipsum dolor sit amet, consectetur
+          adipisicing elit. Ab culpa sint tenetur aut, temporibus cumque
+          dignissimos voluptatum ipsam, iste vitae quis quaerat totam deserunt
+          harum repudiandae excepturi beatae. Repellat, cupiditate.
+        </p>
+        <p class="font-semibold text-blue-800 text-lg">
+          Hier zeige ich dir wie du dein Baby:
+        </p>
+        <nav class="underline text-blue-900 text-opacity-75">
+          <ul class="list-disc list-inside">
+            <li><a href="#here1">Richtig hochnimmst und wieder hinlegst</a></li>
+            <li><a href="#here2">Richtig trägst</a></li>
+            <li><a href="#here3">Richtig wickelst</a></li>
+          </ul>
+        </nav>
+        <img
+          class="rounded-lg"
+          src="./michal-bar-haim-RM9yEZLoJSc-unsplash.jpg"
+          alt="Cute blue-eyed baby hiding under blanket"
+        />
+        <h3 class="text-blue-800 text-3xl font-semibold pt-4">
+          Aber Warum ist das wichtig?
+        </h3>
+        <div class="font-sans text-gray-800">
+          <p class="pb-4">
+            Lorem ipsum, dolor sit amet consectetur adipisicing elit. At, ab
+            sequi. Id quia dolor ut ratione possimus quasi ducimus placeat
+            numquam animi:
+          </p>
+          <ul class="list-disc list-inside pb-4">
+            <li>lorem ipsum, dolor sit amet..</li>
+            <li>lorem ipsum, dolor sit amet..</li>
+            <li>lorem ipsum, dolor sit amet..</li>
+            <li>lorem ipsum, dolor sit amet..</li>
+            <li>lorem ipsum, dolor sit amet..</li>
+          </ul>
+        </div>
+      </div>
+      <div class="bg-blue-100 mx-5 p-6 rounded-lg">
+        <h3 class="text-blue-800 text-3xl font-semibold pb-4">
+          Und so wirds gemacht!
+        </h3>
+        <dl class="space-y-8">
+          <div class="space-y-2 p-2">
+            <dt class="text-blue-800 text-2xl font-semibold" id="here1">
+              Baby richtig hochnehmen und hinlegen
+            </dt>
+            <dd class="font-sans text-gray-800">
+              Lorem ipsum dolor sit amet, consectetur adipisicing elit. Impedit
+              repellat a rem. Quasi atque quo consequatur eligendi cum ducimus
+              corporis unde, quia fugit tenetur omnis, magnam soluta in fuga
+              mollitia!
+            </dd>
+            <img
```

Comment:
soll das Bild mit in das dl Element? Wenn nicht, müsste das dl Element nicht vorher schließen und danach wieder eins eröffnet werden?